### PR TITLE
some misspell

### PR DESCRIPTION
**Pull request by hwdef**
```shell
# edward @ edward-pc in ~/code/cpp/ncnn on git:master o [23:36:05]
$ misspell .
docs/Home.md:123:1: "compability" is a misspelling of "compatibility"
docs/how-to-use-and-FAQ/build-minimal-library.md:74:23: "transfering" is a misspelling of "transferring"
docs/how-to-use-and-FAQ/quantized-int8-inference.md:26:53: "substract" is a misspelling of "subtract"
docs/how-to-use-and-FAQ/use-ncnn-with-alexnet.md:47:136: "distrubute" is a misspelling of "distribute"
docs/how-to-use-and-FAQ/use-ncnn-with-alexnet.md:88:44: "substracted" is a misspelling of "subtracted"
docs/how-to-build/how-to-build.md:46:300: "propietary" is a misspelling of "proprietary"
examples/squeezenet_c_api.cpp:39:13: "substract" is a misspelling of "subtract"
examples/yolact.cpp:193:48: "compatability" is a misspelling of "compatibility"
examples/synset_words.txt:13:42: "mexicanus" is a misspelling of "mexicans"
examples/synset_words.txt:123:69: "americanus" is a misspelling of "americans"
examples/synset_words.txt:296:49: "americanus" is a misspelling of "americans"
examples/synset_words.txt:296:70: "americanus" is a misspelling of "americans"
python/ncnn/model_zoo/yolact.py:216:51: "compatability" is a misspelling of "compatibility"
python/tests/test_mat.py:576:9: "substract" is a misspelling of "subtract"
src/c_api.h:107:26: "substract" is a misspelling of "subtract"
src/c_api.cpp:353:14: "substract" is a misspelling of "subtract"
src/c_api.cpp:355:17: "substract" is a misspelling of "subtract"
python/src/main.cpp:500:5: "substract" is a misspelling of "subtract"
src/layer/arm/convolution_1x1_int8.h:40:20: "shrinked" is a misspelling of "shrunk"
src/layer/arm/convolution_1x1_int8.h:82:42: "shrinked" is a misspelling of "shrunk"
src/layer/arm/convolution_1x1_pack1to4_int8.h:40:20: "shrinked" is a misspelling of "shrunk"
src/layer/arm/convolution_1x1_pack1to4_int8.h:82:51: "shrinked" is a misspelling of "shrunk"
src/layer/arm/convolution_1x1_pack4.h:40:20: "shrinked" is a misspelling of "shrunk"
src/layer/arm/convolution_1x1_pack4.h:64:43: "shrinked" is a misspelling of "shrunk"
src/layer/arm/convolution_1x1_pack4_fp16s.h:667:20: "shrinked" is a misspelling of "shrunk"
src/layer/arm/convolution_1x1_pack4_fp16s.h:716:50: "shrinked" is a misspelling of "shrunk"
src/layer/arm/convolution_1x1_pack4to8_fp16s.h:304:20: "shrinked" is a misspelling of "shrunk"
src/layer/arm/convolution_1x1_pack4to8_fp16s.h:353:53: "shrinked" is a misspelling of "shrunk"
src/layer/arm/convolution_1x1_pack4_bf16s.h:40:20: "shrinked" is a misspelling of "shrunk"
src/layer/arm/convolution_1x1_pack4_bf16s.h:89:49: "shrinked" is a misspelling of "shrunk"
src/layer/arm/convolution_1x1_pack8to1_int8.h:40:20: "shrinked" is a misspelling of "shrunk"
src/layer/arm/convolution_1x1_pack8to1_int8.h:89:51: "shrinked" is a misspelling of "shrunk"
src/layer/arm/convolution_1x1_pack8_fp16s.h:40:20: "shrinked" is a misspelling of "shrunk"
src/layer/arm/convolution_1x1_pack8_fp16s.h:89:50: "shrinked" is a misspelling of "shrunk"
src/layer/arm/convolution_1x1_pack4to1.h:1872:20: "shrinked" is a misspelling of "shrunk"
src/layer/arm/convolution_1x1_pack4to1.h:1896:46: "shrinked" is a misspelling of "shrunk"
src/layer/arm/convolution_1x1_pack8to1_fp16s.h:761:20: "shrinked" is a misspelling of "shrunk"
src/layer/arm/convolution_1x1_pack8to1_fp16s.h:810:53: "shrinked" is a misspelling of "shrunk"
src/layer/arm/convolution_1x1_pack8to4_int8.h:40:20: "shrinked" is a misspelling of "shrunk"
src/layer/arm/convolution_1x1_pack8to4_int8.h:89:51: "shrinked" is a misspelling of "shrunk"
src/layer/arm/convolution_1x1_pack4to1_bf16s.h:2191:20: "shrinked" is a misspelling of "shrunk"
src/layer/arm/convolution_1x1_pack4to1_bf16s.h:2240:52: "shrinked" is a misspelling of "shrunk"
src/layer/arm/convolution_1x1_pack8to4_fp16s.h:785:20: "shrinked" is a misspelling of "shrunk"
src/layer/arm/convolution_1x1_pack8to4_fp16s.h:834:53: "shrinked" is a misspelling of "shrunk"
src/layer/arm/neon_mathfun.h:177:19: "substract" is a misspelling of "subtract"
src/layer/arm/neon_mathfun_fp16s.h:231:19: "substract" is a misspelling of "subtract"
src/layer/arm/neon_mathfun_fp16s.h:292:19: "substract" is a misspelling of "subtract"
src/layer/mips/mips_mathfun.h:157:19: "substract" is a misspelling of "subtract"
src/layer/riscv/convolution_1x1_packn.h:43:20: "shrinked" is a misspelling of "shrunk"
src/layer/riscv/convolution_1x1_packn.h:67:42: "shrinked" is a misspelling of "shrunk"
src/layer/riscv/convolution_1x1_packnto1_fp16s.h:43:20: "shrinked" is a misspelling of "shrunk"
src/layer/riscv/convolution_1x1_packnto1_fp16s.h:67:52: "shrinked" is a misspelling of "shrunk"
src/layer/riscv/convolution_1x1_packn_fp16s.h:43:20: "shrinked" is a misspelling of "shrunk"
src/layer/riscv/convolution_1x1_packn_fp16s.h:67:49: "shrinked" is a misspelling of "shrunk"
src/layer/riscv/convolution_1x1_packnto1.h:43:20: "shrinked" is a misspelling of "shrunk"
src/layer/riscv/convolution_1x1_packnto1.h:67:45: "shrinked" is a misspelling of "shrunk"
src/layer/riscv/rvv_mathfun_fp16s.h:138:23: "substract" is a misspelling of "subtract"
src/layer/riscv/rvv_mathfun.h:138:23: "substract" is a misspelling of "subtract"
src/layer/x86/convolution_1x1_pack4.h:379:20: "shrinked" is a misspelling of "shrunk"
src/layer/x86/convolution_1x1_pack4.h:403:42: "shrinked" is a misspelling of "shrunk"
src/layer/x86/convolution_1x1_pack8to4_int8.h:40:20: "shrinked" is a misspelling of "shrunk"
src/layer/x86/convolution_1x1_pack8.h:977:20: "shrinked" is a misspelling of "shrunk"
src/layer/x86/convolution_1x1_pack8to4_int8.h:64:50: "shrinked" is a misspelling of "shrunk"
src/layer/x86/convolution_1x1_pack8_fp16.h:1012:20: "shrinked" is a misspelling of "shrunk"
src/layer/x86/convolution_1x1_pack8_fp16.h:1036:47: "shrinked" is a misspelling of "shrunk"
src/layer/x86/convolution_1x1_pack8.h:1001:42: "shrinked" is a misspelling of "shrunk"
src/layer/x86/sse_mathfun.h:253:19: "substract" is a misspelling of "subtract"
src/mat.cpp:789:10: "substract" is a misspelling of "subtract"
src/mat.cpp:795:11: "substract" is a misspelling of "subtract"
src/mat.cpp:833:11: "substract" is a misspelling of "subtract"
src/mat.h:274:7: "substract" is a misspelling of "subtract"
src/mat.h:275:9: "substract" is a misspelling of "subtract"
tools/darknet/README.md:40:6: "Remeber" is a misspelling of "Remember"
tools/keras/readme.md:7:77: "maintaince" is a misspelling of "maintained"
tools/darknet/darknet2ncnn.cpp:959:18: "Remeber" is a misspelling of "Remember"
tools/mlir/tf_types.cc:296:61: "compatibilty" is a misspelling of "compatibility"
tools/caffe/caffe.proto:434:46: "substract" is a misspelling of "subtract"
tools/caffe/caffe.proto:1118:67: "explicitely" is a misspelling of "explicitly"
tools/caffe/caffe.proto:1661:26: "paramters" is a misspelling of "parameters"
tools/onnx/onnx.proto:132:24: "accomodate" is a misspelling of "accommodate"
```

just fix a part of

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/ncnn?pullRequest=3046) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/ncnn?pullRequest=3046) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/ncnn?pullRequest=3046) it.</sub>

File: python/src/main.cpp [hunk 1]
```diff
@@ -497,7 +497,7 @@ PYBIND11_MODULE(ncnn, m)
     },
     py::arg("array"), py::arg("type"), py::arg("w"), py::arg("h"), py::arg("stride"), py::arg("roix"), py::arg("roiy"), py::arg("roiw"), py::arg("roih"), py::arg("target_width"), py::arg("target_height"), py::arg("allocator") = nullptr)
     .def(
-    "substract_mean_normalize", [](Mat& mat, std::vector<float>& mean, std::vector<float>& norm) {
+    "subtract_mean_normalize", [](Mat& mat, std::vector<float>& mean, std::vector<float>& norm) {
```
Comment (nihui) [inline python/src/main.cpp:500]: do not change function name

**Review by nihui — CHANGES_REQUESTED** [review]

**Comment by hwdef** [comment]
use tools:

https://github.com/client9/misspell

File: python/src/main.cpp (hunk 1 above)
Comment (hwdef) [reply python/src/main.cpp]: done

File: docs/how-to-use-and-FAQ/quantized-int8-inference.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ find images/ -type f > imagelist.txt
 ./ncnn2table mobilenet-opt.param mobilenet-opt.bin imagelist.txt mobilenet.table mean=[104,117,123] norm=[0.017,0.017,0.017] shape=[224,224,3] pixel=BGR thread=8 method=kl
 ```
 
-* mean and norm are the values you passed to ```Mat::substract_mean_normalize()```
+* mean and norm are the values you passed to ```Mat::subtract_mean_normalize()```
```
Comment (nihui) [inline docs/how-to-use-and-FAQ/quantized-int8-inference.md:26]: set subtract_mean_normalize as an exception ?

**Review by nihui — CHANGES_REQUESTED** [review]

File: docs/how-to-use-and-FAQ/quantized-int8-inference.md (hunk 2 above)
Comment (hwdef) [reply docs/how-to-use-and-FAQ/quantized-int8-inference.md]: ok, keep the old.

**Comment by nihui** [comment]
Thanks for your contribution !

**Pull request merged**